relay-pii/src/processor.rs

```diff
@@ -200,23 +200,10 @@ impl<'a> Processor for PiiProcessor<'a> {
         replay.process_child_values(self, state)?;
         Ok(())
     }
-
-    fn process_event(
-        &mut self,
-        event: &mut Event,
-        _meta: &mut Meta,
-        state: &ProcessingState<'_>,
-    ) -> ProcessingResult {
-        scrub_graphql(event);
-
-        event.process_child_values(self, state)?;
-
-        Ok(())
-    }
 }
 
 /// Scrubs GraphQL variables from the event.
-fn scrub_graphql(event: &mut Event) {
+pub fn scrub_graphql(event: &mut Event) {
```

Comment:
Since it's not used in the processor directly we might consider moving it elsewhere, thoughts?